sync/fetch_queue.go

```diff
@@ -9,29 +9,29 @@ import (
 	"sync"
 )
 
-type FetchPoetProofFunc func(poetProofRef []byte) error
-type SValidateAtxFunc func(atx *types.ActivationTx) error
-type CheckLocalFunc func(ids []types.Hash32) (map[types.Hash32]Item, map[types.Hash32]Item, []types.Hash32)
+type fetchPoetProofFunc func(poetProofRef []byte) error
+type sValidateAtxFunc func(atx *types.ActivationTx) error
+type checkLocalFunc func(ids []types.Hash32) (map[types.Hash32]item, map[types.Hash32]item, []types.Hash32)
 
-type Item interface {
+type item interface {
 	Hash32() types.Hash32
 	ShortString() string
 }
 
 type fetchJob struct {
-	items []Item
+	items []item
 	ids   []types.Hash32
 }
 
 //todo make the queue generic
 type fetchQueue struct {
```

Comment:
I guess some commenting about how `fetchQueue` works can be useful even though it's not exported

Reply:
true, im gonna do more doc pr's after this one